README.md

```diff
@@ -90,6 +93,19 @@ The installer will download [config.xml](https://raw.githubusercontent.com/mandi
 .\install.ps1 -customConfig "https://raw.githubusercontent.com/mandiant/flare-vm/main/config.xml"
 ```
 
+#### Taskbar Layout
+The installer will use [CustomStartLayout.xml](https://raw.githubusercontent.com/mandiant/flare-vm/main/CustomStartLayout.xml) from the FLARE-VM repository. This file contains the default taskbar layout. You may use your own configuration by specifying the CLI-argument `-customLayout` and providing a local file path or URL to your `CustomStartLayout.xml` file. For example:
+
+```
+.\install.ps1 -customLayout "https://raw.githubusercontent.com/mandiant/flare-vm/main/CustomStartLayout.xml"
+```
+
+##### Things to Consider:
```

Comment:
Should we also add here an item about how to ping items that run with admin rights (consider creating a shortcut...)